web-app/src/components/button/button.tsx

```diff
@@ -63,7 +64,15 @@ const Button = styled.button<ButtonExtraProps>`
 		text-align: center;
 		white-space: nowrap;
 		vertical-align: middle;
-		padding: 0.9rem 3rem;
+		padding:${(props) => {
+		if (props.size === "small") {
+			return "0.9rem 3rem";
+		} else if (props.size === "small") {
```

Comment:
Is this a mistake or why did you replicate this condition? Maybe you meant to create conditions for size medium and large.

Reply:
I tried but then I realize button size would be needed to change based on screen size . So didn't implemented that. Will remove this code.